lib/marked-renderer.js

```diff
@@ -1,5 +1,12 @@
 const { marked } = require('marked')
 
+// Temporary fix for silencing deprecation messages till options are removed
+// in a future version of `marked`
+marked.use({
```

Comment:
I'm happy with the workaround as long as we (definitely) don't need them?

Probably want to configure our renderer only a few lines down

```js
class DesignSystemRenderer extends marked.Renderer {
  constructor () {
    super({
      mangle: false,
      headerIds: false
    })
  }

  // …
}
```

Reply:
We don't use `mangle` at all anywhere on the site, and we don't use `headerIds` within that renderer, but as per your comment below, I'll investigate using the `marked-gfm-heading-id` plugin now.